nautobot/core/tests/test_migration.py

```diff
@@ -0,0 +1,40 @@
+from django.apps import apps
+from django.core.management import call_command
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+
+class NautobotDataMigrationTest(TestCase):
```

Comment:
Eventually we should ship developer docs on this because :chef's_kiss: this is super powerful. 🚀